### PR TITLE
docs: update kong readme

### PR DESCRIPTION
**Pull request by fishkerez**
https://portshift.atlassian.net/browse/MAN-8162

### CODE REVIEW

File: plugins/gateway/kong/README.md [hunk 1]
```diff
@@ -1,5 +1,75 @@
...
+       cd ~/go/src/github.com/apiclarity/apiclarity/plugins/gateway/kong
+       
+       KONG_GATEWAY_DEPLOYMENT_NAME=ingress-kong \
+       KONG_GATEWAY_DEPLOYMENT_NAMESPACE=kong \
+       KONG_GATEWAY_INGRESS_NAME=catalogue \
+       KONG_GATEWAY_INGRESS_NAMESPACE=sock-shop \
+       UPSTREAM_TELEMETRY_HOST_NAME=apiclarity-apiclarity.apiclarity:9000 \
+       deploy/deploy.sh
+    ```   
+    * Note: If you installed using helm, the deployment name might be different. Please change the KONG_GATEWAY_DEPLOYMENT_NAME env var accordingly.
```
Comment (FrimIdan) [inline plugins/gateway/kong/README.md:61]: `If you installed using helm` - installed kong or APIClarity?

File: plugins/gateway/kong/README.md [hunk 2]
```diff
@@ -1,5 +1,75 @@
...
+             - path: /
+               pathType: Prefix
+               backend:
+                 service:
+                   name: catalogue
+                   port:
+                     number: 80
+       EOF
+    ```   
+5. Deploy APIClarity Kong Plugin:
```
Comment (FrimIdan) [inline plugins/gateway/kong/README.md:50]: I think it should be more like https://github.com/apiclarity/apiclarity/tree/master/plugins/gateway/tyk#installation-using-a-pre-built-image to include also the helm installation

File: plugins/gateway/kong/README.md [hunk 3]
```diff
@@ -1,5 +1,75 @@
...
+
+If you just want to experience with it, and you don't have kong, you can take these few quick steps that will help you to quickly setup a running environment:
+1. Deploy sock-shop app:
+    ```shell
+       kubectl create namespace sock-shop
+       
+       kubectl apply -f https://raw.githubusercontent.com/microservices-demo/microservices-demo/master/deploy/kubernetes/complete-demo.yaml
+    ```
+2. Deploy Kong:
+    - Using helm3:
```
Comment (FrimIdan) [inline plugins/gateway/kong/README.md:15]: from https://docs.konghq.com/kubernetes-ingress-controller/2.2.x/deployment/k4k8s/#helm the following is also needed.
```
$ helm repo add kong https://charts.konghq.com
$ helm repo update
```

File: plugins/gateway/kong/README.md [hunk 4]
```diff
@@ -1,5 +1,75 @@
...
+    ```shell
+       helm install kong/kong --generate-name --set ingressController.installCRDs=false
+    ```
+    - Using kubectl:
+    ```shell
+       kubectl create namespace kong
```
Comment (FrimIdan) [inline plugins/gateway/kong/README.md:21]: no need

File: plugins/gateway/kong/README.md [hunk 5]
```diff
@@ -1,5 +1,75 @@
...
+    ```shell
+       export PROXY_IP=$(kubectl get -o jsonpath="{.status.loadBalancer.ingress[0].ip}" service -n kong kong-proxy)
+    ```
+    * Note: If you installed using helm, the service name might be different.    
+7. Run Traffic:
+    ```shell
+       curl -H 'content-type: application/json' -H 'accept: application/json;charset=UTF-8' $PROXY_IP/catalogue
+       curl -H 'content-type: application/json' -H 'accept: application/json;charset=UTF-8' $PROXY_IP/catalogue/size
+       curl -H 'content-type: application/json' -H 'accept: application/json;charset=UTF-8' $PROXY_IP/tags
+    ```
```
Comment (FrimIdan) [inline plugins/gateway/kong/README.md:72]: please add a cleanup step to delete/uninstall all added components

File: plugins/gateway/kong/README.md [hunk 6]
```diff
@@ -1,5 +1,75 @@
 ## APIClarity Kong plugin
 
+### Prerequisite
+* APIClarity backend is running
+* Kong gateway is running in your K8s cluster, and configured correctly.
```
Comment (akpsgit) [inline plugins/gateway/kong/README.md:5]: I think that "correctly" is not very clear

File: plugins/gateway/kong/README.md [hunk 7]
```diff
@@ -1,5 +1,75 @@
...
+
+If you just want to experience with it, and you don't have kong, you can take these few quick steps that will help you to quickly setup a running environment:
```
Comment (akpsgit) [inline plugins/gateway/kong/README.md:7]: If you just want to try it out, and you don't have kong installed, you can follow these few short steps that will help you to quickly setup a running environment:

File: plugins/gateway/kong/README.md [hunk 8]
```diff
@@ -1,5 +1,75 @@
...
+       KONG_GATEWAY_DEPLOYMENT_NAMESPACE=kong \
+       KONG_GATEWAY_INGRESS_NAME=catalogue \
+       KONG_GATEWAY_INGRESS_NAMESPACE=sock-shop \
+       UPSTREAM_TELEMETRY_HOST_NAME=apiclarity-apiclarity.apiclarity:9000 \
+       deploy/deploy.sh
+    ```   
+    * Note: If you installed using helm, the deployment name might be different. Please change the KONG_GATEWAY_DEPLOYMENT_NAME env var accordingly.    
+6. Get LoadBalacner IP:
+    ```shell
+       export PROXY_IP=$(kubectl get -o jsonpath="{.status.loadBalancer.ingress[0].ip}" service -n kong kong-proxy)
```
Comment (akpsgit) [inline plugins/gateway/kong/README.md:64]: in EKS, ist's a domain and not IP, maybe worth adding a comment

File: plugins/gateway/kong/README.md (hunk 2 above)
Comment (fishkerez) [reply plugins/gateway/kong/README.md:50]: Its mentions in later section. here it is just a quick installation

File: plugins/gateway/kong/README.md [hunk 9]
```diff
@@ -1,5 +1,92 @@
...
+    ```
+7. Cleanup:
+    
+    1. Delete kong installation:
+        ```shell
+           helm -n kong uninstall kong 
+        ```   
+          Or if not installed with helm:
+        ```shell
+            kubectl delete ns kong
```
Comment (FrimIdan) [inline plugins/gateway/kong/README.md:84]: `kubectl delete -f https://bit.ly/kong-ingress-dbless` is better since there are non namespace objects applied.

**Comment by FrimIdan** [comment]
LGTM

**Review by akpsgit — APPROVED** [review]

**Pull request merged**